tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -581,6 +581,12 @@ def docomplicated(suptitle=None):
 #
 # * There are small differences in how the backends handle rendering fonts,
 #   so the results will not be pixel-identical.
+#
+# * A :class:`~matplotlib.patches.ConnectionPatch` across two
+#   :class:`~matplotlib.axes.Axes` must be removed from the layout
+#   using :meth:`~matplotlib.artist.Artist.set_in_layout` before
+#   calling :meth:`~matplotlib.axes.Axes.add_artist`. See
+#   :class:`~matplotlib.patches.ConnectionPatch` for more information.
```

Comment:
In principle this applies to just any artist that uses axes coordinates and is added to an axes. So similar problems are expected for annotations or annotation boxes in axes coordinates, ax{h,v}lines if they overflow the axes boundary, legends (which are actually mentionned someplace earlier in this tutorial), etc. 

Maybe there is a way to formulate this more generally and name the connection patch as an example?

Reply:
Yeah sure. I updated the constrained_layout tutorial to apply to all artists. Let me know what you think.

Reply:
Contentwise I'd be fine with that. I'll leave it to a native speaker to check if that sentence makes sense grammatically. Probably, @jklymak, the original author of that tutorial would like to have a look at it anyways.